### PR TITLE
Support iperf

### PR DESCRIPTION
**Pull request by saimonation**

### CODE REVIEW

File: cterasdk/edge/network.py
Comment (ygalblum) [inline cterasdk/edge/network.py:161]: defaults to 120

**Review by ygalblum — APPROVED** [review]
I'm approving with the minor typo. Once fixed, just merge, no need for another review

File: cterasdk/edge/network.py
Comment (saimonation) [reply cterasdk/edge/network.py]: Thanks! I changed it last minute :)

**Pull request merged**
